### PR TITLE
DNS Data set documentation [See Spot-218]

### PR DESCRIPTION
**Pull request by brandon-edwards**
Put data set on S3 and document with mark down file in the Apache repo.

### CODE REVIEW

File: spot-ml/DATA_SAMPLE.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+# DNS Labeled Data Set
+
+An IXIA BreakingPoint box was used to simulate both normal and attack (DNS tunnelling) DNS traffic. The resulting pcaps were obtained and fields relevant to Apache Spot (incubating) were injested. The attacks can be differentiated from the normal activity due to codes that were inserted into the Transaction ID field (upon ingestion: ‘dns_id’) which identifies either the fact that the traffic was normal or identifies the specific DNS tunneling activity being used. We provide the data schema as well as the location and specifications of the data within Amazon-S3. Information is also provided for how to interpret the dns_id field.
```
Comment (NathanSegerlind) [inline spot-ml/DATA_SAMPLE.md]: injested => ingested

File: spot-ml/DATA_SAMPLE.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+
+## Data Schema
+
+The schema for this data includes one field (called 'dns_id') in addition to what is usually used for DNS data in Apache Spot (incubating). The schema is as follows:
+
+
+| Name         | Type      |
```
Comment (NathanSegerlind) [inline spot-ml/DATA_SAMPLE.md]: a column explaining the meaning of each field would be helpful

File: spot-ml/DATA_SAMPLE.md (hunk 1 above)
Comment (brandon-edwards) [reply spot-ml/DATA_SAMPLE.md]: indeed

File: spot-ml/DATA_SAMPLE.md (hunk 2 above)
Comment (brandon-edwards) [reply spot-ml/DATA_SAMPLE.md]: on it

**Comment by vgonzale78** [comment]
+1

**Pull request merged**